examples/routing_service/routing_service_file_adapter/cpp/README.md

```diff
@@ -0,0 +1,79 @@
+# Example Code :Routing Service C++ Adapter
+
+## Example Description
+
+This example shows how to implement a custom `Routing Service Adapter plug-in`,
+build it into a shared library and load it with RoutingService.
+
+In this concrete example, we show how a simple adapter to import/export data from/to files using RTI Routing Service.
+
+The code in this directory provides the following components:
+
+- `FileAdapter` implements the plugin that is loaded by *RTI Routing Service*. It is responsible to create and delete connections.
+- `FileConnection` implements a connection. This component is responsible of the creation and deletion of `StreamReaders` and `StreamWriters`.
+- `FileDiscoveryStreamReader` implements the logic necessary to propagate information about newly discovered streams to RTI Routing Service.
+- `FileStreamReader` implements an `StreamReader` that reads sample information from a CSV file.
```

Comment:
why reading from CSV and writing to JSON? should we be symmetric? or even better, have an option to decide the format?

Reply:
We now read from and write in a CSV format for ShapeType.